models.rb

```diff
@@ -36,7 +36,7 @@ class MenuItem
 	property :description,			Text, required: true
 	
 	has n, :reviews
-	has n, :lunch_menu_items
+	has n, :lunch_menu_items, constraint: :destroy
```

Comment:
Let's make sure there aren't other associations that are suffering from this same issue.